users/models.py

```diff
@@ -133,3 +133,27 @@ class Meta:
 
     def __str__(self):
         return f"UserCampaignRegistration {self.user.username}({self.key})"
+
+
+class UserActivityLog(models.Model):
```

Comment:
I have few concerns with this table growing indefinitely, given we don't have any mechanism for removing old entries:
- how fast does it grow given the traffic we have: did you do any rough calculations on how much storage it will take? 
- essentially we could slowly be DDOSed - somebody can just run a few infinite loops making requests on behalf of a user which will just add entries to the DB.
- this makes GDPR more difficult than it is today, and we apparently need to become compliant very very soon (following weeks). AFICT (though not expert in GDPR), must not store user info that is not justified for the functioning of the system, and not for longer than required.